src/lib/components/composites/settings/project-settings/review/KeywordSettings.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import SettingsSection from "$lib/components/composites/settings/SettingsSection.svelte";
+    import TagsInput from "svelte-unstyled-tags";
+    import { toast } from "svelte-sonner";
+    import { onMount } from "svelte";
+
+    interface Props {
+        projectId: string;
+        maximumAmountOfTags: number;
```

Comment:
"amount" is always for things that cannot be assigned to single units, e.g., liquids. For something that can be distinguished into units, we use "number".
```suggestion
        maximumNumberOfTags: number;
```

Reply:
Done